packages/web/src/pages/pick-winners-page/PickWinnersPage.tsx

```diff
@@ -0,0 +1,212 @@
+  FilterButton,
+  Flex,
+  IconButton,
+  IconMinus,
+  IconPlus,
+  IconCheck,
+  Paper,
+  Text,
+  useTheme
+} from '@audius/harmony'
+import { useDispatch } from 'react-redux'
+import { useParams } from 'react-router'
+
+import { Header } from 'components/header/desktop/Header'
+import { TanQueryLineup } from 'components/lineup/TanQueryLineup'
+import { Page } from 'components/page/Page'
+import { useUpdateSearchParams } from 'pages/search-page/hooks'
+
+import { usePickWinnersPageParams } from './hooks'
+
+const PICK_WINNERS_PAGE_SIZE = 10
+const MAX_WINNERS = 5
+
+export const PickWinnersPage = () => {
+  const dispatch = useDispatch()
+  const { color, motion } = useTheme()
+  const { handle, slug } = useParams<{ handle: string; slug: string }>()
+  const { data: originalTrack } = useTrackByPermalink(
+    handle && slug ? `/${handle}/${slug}` : null
+  )
+
+  const updateSortParam = useUpdateSearchParams('sortMethod')
+  const updateIsCosignParam = useUpdateSearchParams('isCosign')
+
+  const { sortMethod, isCosign } = usePickWinnersPageParams()
+  const {
+    data,
+    count,
+    isFetching,
+    isPending,
+    isError,
+    hasNextPage,
+    play,
+    pause,
+    loadNextPage,
+    isPlaying,
+    lineup
+  } = useRemixes({
+    trackId: originalTrack?.track_id,
+    sortMethod,
+    isCosign,
+    isContestEntry: true
+  })
+
+  const pageHeader = (
+    <Header
+      primary={messages.pickWinnersTitle}
+      showBackButton
+      rightDecorator={
+        <Button size='small' disabled>
+          {messages.finalizeWinners}
+        </Button>
+      }
+    />
+  )
+
+  const [winners, setWinners] = useState<ID[]>([])
+
+  const addIdToWinners = useCallback(
+    (id: ID) => {
+      if (winners.includes(id)) return
+      if (winners.length >= MAX_WINNERS) {
+        dispatch(toast({ content: messages.maxWinnersReached }))
+        return
+      }
+
+      setWinners([...winners, id])
+    },
+    [winners, dispatch]
+  )
+
+  const removeIdFromWinners = useCallback(
+    (id: ID) => {
+      if (!winners.includes(id)) return
+      setWinners(winners.filter((winnerId) => winnerId !== id))
+    },
+    [winners]
+  )
+
+  const tileAdornment = (elementId: ID, index: number) => {
+    const Icon = winners.includes(elementId) ? IconCheck : IconPlus
+    return (
+      <Paper
+        p='xs'
+        shadow='flat'
+        borderRadius='circle'
+        css={{
+          position: 'absolute',
+          top: '50%',
+          left: 0,
+          backgroundColor: color.background.white,
+          transform: 'translate(-50%, -50%)',
+          transition: motion.hover,
+          '&:hover': {
+            backgroundColor: color.neutral.n100,
+            transform: 'translate(-50%, -50%) scale(1.1)'
+          },
+          '&:active': {
+            backgroundColor: color.neutral.n150,
+            transform: 'translate(-50%, -50%) scale(0.98)'
+          }
+        }}
+        onClick={() => addIdToWinners(elementId)}
```

Comment:
nit: useCallback-ify

Reply:
normally would, but cant for this bc its in a function